### PR TITLE
Fix podman stats based on QE feedback

### PR DESCRIPTION
**Pull request by umohnani8**
QE found issues with formatting the go template and
the man page was lacking information.
Changed the format of the output to match latest docker.

Signed-off-by: umohnani8 <umohnani@redhat.com>

### CODE REVIEW

**Comment by umohnani8** [comment]
@rhatdan @mheon @baude PTAL

**Comment by rhatdan** [comment]
LGTM

File: cmd/podman/stats.go [hunk 1]
```diff
@@ -248,13 +253,13 @@ func pidsToString(pid uint64) string {
...
-		Container: stats.ContainerID[:12],
-		ID:        stats.ContainerID,
-		CPUPerc:   floatToPercentString(stats.CPU),
-		MemUsage:  combineHumanValues(stats.MemUsage, stats.MemLimit),
-		MemPerc:   floatToPercentString(stats.MemPerc),
-		NetIO:     combineHumanValues(stats.NetInput, stats.NetOutput),
-		BlockIO:   combineHumanValues(stats.BlockInput, stats.BlockOutput),
-		PIDS:      pidsToString(stats.PIDs),
+		Name:     stats.Name,
+		ID:       stats.ContainerID[:idTruncLength],
```
Comment (TomSweeneyRedHat) [inline cmd/podman/stats.go]: I know it should never happen, but being Irish that just means it's going to happen sooner.  Can we do a len check on stats.ContainerID before this line?  In another review a thought had gone up about making a 'ShortID()" utility function somewhere to safely get a short id.  If that hasn't happened, could you make it so?

File: docs/podman-stats.1.md [hunk 2]
```diff
@@ -33,44 +33,49 @@ Disable streaming stats and only pull the first result, default setting is false
 
 Pretty-print images using a Go template
 
+Valid placeholders for the Go template are listed below:
```
Comment (TomSweeneyRedHat) [inline docs/podman-stats.1.md]: Could you add a template example or two below?  Also the example has a formt=json example, but no mention of jsno anywhere else in the man page.  Can you add that too please?

**Comment by rhatdan** [comment]
Lets make the ShortID function.

File: cmd/podman/stats.go (hunk 1 above)
Comment (umohnani8) [reply cmd/podman/stats.go]: Added shortID() function.

File: docs/podman-stats.1.md (hunk 2 above)
Comment (umohnani8) [reply docs/podman-stats.1.md]: fixed

**Comment by umohnani8** [comment]
Added shortID function and used it everywhere the ID is truncated.

**Comment by rhatdan** [comment]
LGTM

**Comment by rhatdan** [comment]
@TomSweeneyRedHat @baude @mheon PTAL

File: cmd/podman/common.go [hunk 3]
```diff
@@ -88,6 +91,13 @@ func splitCamelCase(src string) string {
 	return strings.Join(entries, " ")
 }
 
+func shortID(id string) string {
+	if len(id) > idTruncLength {
+		return id[:idTruncLength]
+	}
+	return id
+}
+
```
Comment (TomSweeneyRedHat) [inline cmd/podman/common.go]: TY!

**Comment by TomSweeneyRedHat** [comment]
Changes LGTM, but Red Hat CI isn't happy.

**Comment by mheon** [comment]
LGTM

**Comment by umohnani8** [comment]
bot retest this please

**Comment by rhatdan** [comment]
@rh-atomic-bot r+

**Pull request closed without merging**